manifest.json

```
@@ -1,7 +1,7 @@
 {
   "name": "b2b-organizations-graphql",
   "vendor": "vtex",
-  "version": "2.0.0",
+  "version": "1.1.0",
```

lets not touch this version. this should be done only during the release process, which is made automatically when merging the PR.
```suggestion
  "version": "2.0.0",
```